static-assets/css/style.css

```diff
@@ -74,15 +74,8 @@ VIDEO BACKGROUND CSS
 -----------------------------------------*/
 .header-video {
   overflow: hidden;
-}
-
-.header-video iframe,
-.header-video video {
-  position: absolute;
-  top: 0;
-  bottom: 0;
-  left: 0;
-  right: 0;
+  width: 100%;
+  height: 100%;
```

Comment:
```suggestion
  height: 95vh;
  @media (min-width: 850px) {
    height: 100vh;
  }
```

and maybe add

Reply:
@rart `maybe add` was something missing in this message?

Reply:
no, just the code suggestion